schoolyourself/schoolyourself.py

```diff
@@ -125,7 +126,8 @@ def get_student_id(self):
 
     def resource_string(self, path):
       """Handy helper for getting resources from our kit."""
-      data = pkg_resources.resource_string(__name__, path)
+      ref = importlib.resources.files("schoolyourself").joinpath(path)
```

Comment:
Perfer to use xblock.utils method
Follow: https://github.com/openedx/xblock-in-video-quiz/pull/212/files#diff-cf5d4b07c1f2d28759f30b933fbdacaebf074f45fe08f1890b1fec6a9a049148R39

Reply:
[resolved](https://github.com/openedx/schoolyourself-xblock/pull/138/commits/9e1d2d2b5d4f439cf38eb1e9e994f1224f8b188c)